test/skimNtuple_HHbtag.cpp

```diff
@@ -2707,8 +2707,10 @@ int main (int argc, char** argv)
 
 	  float leg1pt  = tlv_firstLepton.Pt();
 	  float leg1eta = TMath::Abs(tlv_firstLepton.Eta());
+	  float leg1eta_SC = theBigTree.daughters_SCeta->at(firstDaughterIndex);
```

Comment:
Add a comment on why this is needed, otherwise it might not be clear in a few months.